media/css/firefox/new/compare.scss

```diff
@@ -5,12 +5,11 @@
 $font-path: '/media/fonts';
 $image-path: '/media/protocol/img';
 
-@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/includes/lib";
-@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/components/button";
-@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/components/download-button";
```

Comment:
I'm removing the `download-button` component here, as it doesn't seem to be used but please check (the page is using custom styles).